### DISCUSSION TITLE
Glow Colors when eating berries

### DISCUSSION THREAD

**Discussion by itsMarkiS** [PurpurMC/Purpur#616] [__Dismissed__] [answered]
would be nice if we could change the glow color when eating glow berries, also if it could have a "rainbow glow feature" where it changes color depending on the time of the berry, or even displaying a random color per glow berry eaten

**Comment by YouHaveTrouble** [comment]
Glow color is based on scoreboard team. This could mess up plugins that use teams, so scoreboards, other plugins that handle glowing, collision rules etc.

**Reply by itsMarkiS** [reply]
yeah but if possible, would be cool to add something like "overlap scoreboard - true or false" so if you set to false it will be according to the teams, and if set to true you can define the color

**Reply by YouHaveTrouble** [reply]
just the fact of it requiring messing with teams is a no-no imo.

**Reply by RhythmicSys** [reply]
Gonna just reply in this thread to avoid double messages. But gonna agree here, dealing with teams would require too much intervention with base game I feel. Definitely more plugin than software like purpur

**Reply by itsMarkiS** [reply]
oh... understandable thx

**Comment by RhythmicSys** [comment, marked as answer]
Unsure if this would be implementable since that would require placing the user in a team, and would mess with plugins that depend on teams

**Reply by itsMarkiS** [reply]
yeah but if possible, would be cool to add something like "overlap scoreboard - true or false" so if you set to false it will be according to the teams, and if set to true you can define the color
